src/auth.js

```diff
@@ -317,13 +335,14 @@ module.exports = {
     stashHandle,
     login.ensureLoggedIn('/auth/login'),
     server.authorization(authdb.validateClient, (client, user, scope, type, req, done) => {
-      // Auto-approve
+      // Auto-approve for home immer
       if (client.isTrusted) {
         const params = {}
         const origin = new URL(req.redirectURI)
         params.origin = `${origin.protocol}//${origin.host}`
         // express protocol does not include colon
         params.issuer = `https://${domain}`
+        params.scope = ['*']
         return done(null, true, params)
       }
```

Comment:
When logging into your home immer, scope authorization is skipped and all scopes are granted. The easiest way to test this locally is to comment out this entire `isTrusted` block and then you'll get the scoping dialog when logging in (clear your hubs localstorage to trigger re-authorization if you've already got a valid token)